application/templates/_shared/_breadcrumb.html

```diff
@@ -1,14 +1,19 @@
-{%  macro breadcrumb() %}
+{% macro render_breadcrumbs(breadcrumbs) %}
+  {% if breadcrumbs %}
     <div class="grid-row">
-        <div class="column-full">
-            <nav class="" aria-label="You are here:" role="navigation">
-                <ul class='breadcrumbs'>
-                    <li>
-                        <a href="{{ url_for('static_site.index') }}">Ethnicity facts and figures</a>
-                    </li>
-                    {{ caller() }}
-                </ul>
-            </nav>
-        </div>
+      <div class="column-full">
+        <nav class="" aria-label="You are here:" role="navigation">
+          <ol class='breadcrumbs'>
+            {% for breadcrumb in breadcrumbs %}
+            <li>
+                <a href="{{ breadcrumb.url }}"{% if breadcrumb.event %} data-event="{{ breadcrumb.event }}"{% endif %}>{{ breadcrumb.text }}</a>
+            </li>
+            {% endfor %}
+          </ol>
+        </nav>
+      </div>
     </div>
+  {% elif breadcrumbs is not defined %}
+    <p>NEEDS CENTRALISED BREADCRUMBS</p>
```

Comment:
This isn't a thing we would ever want to show on the site?  Doesn't feel like it should be in the template here.  We could either fail hard with an error, or just have the homepage as a default breadcrumb? ie
```
<nav class="" aria-label="You are here:" role="navigation">
                <ul class='breadcrumbs'>
                    <li>
                        <a href="{{ url_for('static_site.index') }}">Ethnicity facts and figures</a>
                    </li>
                </ul>
            </nav>
```

Reply:
This was just for debugging missing templates. Need to remove it, yes. :)